packages/kotti-ui/source/kotti-field-radio-group/KtFieldRadioGroup.vue

```diff
@@ -96,6 +98,15 @@ export default defineComponent<KottiFieldRadioGroup.PropsInternal>({
 
 				forceUpdate()
 			},
+			optionDataTest: (option: KottiFieldRadioGroup.Entry) => {
+				if (option.dataTest) return option.dataTest
+
+				if (Object.keys(field.inputProps).includes('data-test')) {
+					return [field.inputProps['data-test'], option.value].join('.')
+				}
+
+				return undefined
```

Comment:
Is it necessary this return? Not returning anything from a function will default to undefined

Reply:
you're right, just wanted to make it explicit that the data-test would be unbound on html